packages/lesswrong/server/resolvers/bookmarksFeedResolver.ts

```diff
@@ -0,0 +1,165 @@
+import gql from 'graphql-tag';
+import { Bookmarks } from '../collections/bookmarks/collection';
+import { loadByIds } from '../../lib/loaders';
+import { filterNonnull } from '../../lib/utils/typeGuardUtils';
+import { FeedPostMetaInfo, FeedCommentMetaInfo, FeedPostResolverType, FeedCommentsThreadResolverType as UltraFeedCommentsThreadResolverType } from '../../components/ultraFeed/ultraFeedTypes';
+import uniq from 'lodash/uniq';
+import orderBy from 'lodash/orderBy';
+
+export const bookmarksFeedGraphQLTypeDefs = gql`
+  type BookmarksFeedQueryResults {
+    cutoff: Date
+    endOffset: Int!
+    results: [BookmarksFeedEntryType!]
+  }
+  
+  type BookmarksFeedEntryType {
+    type: String!          # 'feedPost' or 'feedCommentThread'
+    feedPost: FeedPost     
+    feedCommentThread: FeedCommentThread 
+    sortKey: Date!         
+  }
+  
+  extend type Query {
+    BookmarksFeed(
+      limit: Int,
+      cutoff: Date,
+      offset: Int,
+    ): BookmarksFeedQueryResults!
+  }
+`
+
+interface FeedCommentsThreadResolverType extends UltraFeedCommentsThreadResolverType {
+  post: Partial<DbPost> | null;
+}
+
+type InternalFeedItem = {
+  type: 'feedPost' | 'feedCommentThread';
+  feedPost: FeedPostResolverType | null;
+  feedCommentThread: FeedCommentsThreadResolverType | null;
+  sortKey: Date;
+};
+
+interface BookmarksFeedQueryResult {
+  cutoff: Date | null;
+  endOffset: number;
+  results: BookmarksFeedEntryType[];
+}
+
+export const bookmarksFeedGraphQLQueries = {
```

Comment:
Wait, is this whole thing just to have a chronological list of bookmarks with a `load more`??

Reply:
you don't need a custom resolver for this!

Reply:
(ignore the rest of my feedback on this file if you can in fact just replace this entire thing with the default `bookmarks` multi resolver 🤦)

Reply:
...yes

The MixedTypeFeed handles having different renderers and resolver handles getting them with required data...I grant that it ends up being a bunch of code

Reply:
I don't think that really makes sense... what extra data are you even getting?  The relevant post/comment (which you can just have a resolver field for)?

Reply:
ok, changed, done